api/healthCheck/index.ts

```diff
@@ -47,26 +44,27 @@ export function getPromises(path): any[] {
             break
         }
     }
-
-    const health = getConfigValue(HEALTH)
-
     if (healthCheckEndpointDictionary[path]) {
+        console.log(healthCheckEndpointDictionary[path])
         healthCheckEndpointDictionary[path].forEach(endpoint => {
-            Promises.push(http.get(health[endpoint]))
+            // TODO: Have health config for this.
+            console.log('healthEndpoints', endpoint)
+            console.log(healthEndpoints()[endpoint])
+            Promises.push(http.get(healthEndpoints()[endpoint]))
         })
     }
     return Promises
 }
 
 export async function healthCheckRoute(req, res) {
-    res.send({ healthState: true })
-    /*try {
+    // res.send({ healthState: true })
```

Comment:
If this is no longer needed, can it be removed?

Reply:
removing